docs/management/persistence.md

```diff
@@ -259,26 +259,35 @@ and starts appending new data into the new file.
 
 ### How I can switch to AOF, if I'm currently using dump.rdb snapshots?
 
-There is a different procedure to do this in version 2.0 and later versions, as you
-can guess it's simpler since Redis 2.2 and does not require a restart at all.
+If you want to enable AOF in a server that is currently using RDB snapshots, you need to convert the data by enabling AOF via CONFIG command on the live server first.
+
+**IMPORTANT:** not following this procedure (e.g. just changing the config and restarting the server) can result in data loss!
 
 **Redis >= 2.2**
 
+Preparations:
+
 * Make a backup of your latest dump.rdb file.
 * Transfer this backup to a safe place.
-* Issue the following two commands:
-* `redis-cli config set appendonly yes`
-* `redis-cli config set save ""`
-* Make sure your database contains the same number of keys it contained.
-* Make sure writes are appended to the append only file correctly.
 
-The first CONFIG command enables the Append Only File persistence.
+Switch to AOF on live database:
 
-The second CONFIG command is used to turn off snapshotting persistence. This is optional, if you wish you can take both the persistence methods enabled.
+* Enable AOF: `redis-cli config set appendonly yes`
+* Optionally disable RDB: `redis-cli config set save ""`
+* Wait for AOF rewrite to finish persisting the data. You can do that by
```

Comment:
This is important to check only if you want to restart the server. It is not necessary to restart the server. Maybe move this to "Before restarting the server" section, or at least mention that it's related to restarting?

Reply:
Good point, updated in 2f202c889f73918a2f377737b015d287ddf3debd